get_involved.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Get Involved
```

Comment:
This PR adds a second `get_involved.md` page.  There is already one at at `pages/get_involved.md`.  This adds another at the root directory.  Pages were collected into `pages/` directory to reduce number of files in root directory.  The `permalink: /get_involved/` header will make sure page exists at root for URLs.